### PR TITLE
[DMN Editor] Fix relationship with property link

### PR DESCRIPTION
**Pull request by danielzhe**
[DMN Editor] Fix relationship with property link
[DMN Editor] Decision Table Rules (IDs are not managed in React)

### CODE REVIEW

File: packages/boxed-expression-component/src/table/BeeTable/BeeTableEditableCellContent.tsx [hunk 1]
```diff
@@ -74,6 +78,12 @@ export function BeeTableEditableCellContent({
     setEditingValue((prev) => (isEditing ? prev : value));
   }, [isEditing, value]);
 
+  useEffect(() => {
+    if (isActive) {
+      beeGwtService?.selectObject(cellId);
+    }
+  }, [isActive, beeGwtService, cellId]);
+
```
Comment (tiagobento) [inline packages/boxed-expression-component/src/table/BeeTable/BeeTableEditableCellContent.tsx:86]: We can move this up to the `BeeTableDefaultCell`.. this way we avoid an extra prop here :)

**Review by tiagobento — CHANGES_REQUESTED** [review]
Very clean code and changes. Nicely done! Left a comment inline for moving the selection logic one level up the component tree :)

File: packages/boxed-expression-component/src/table/BeeTable/BeeTableEditableCellContent.tsx (hunk 1 above)
Comment (danielzhe) [reply packages/boxed-expression-component/src/table/BeeTable/BeeTableEditableCellContent.tsx]: @tiagobento Changes applied!

**Review by tiagobento — APPROVED** [review]
Excellent!

**Pull request merged**